guardian/support-frontend
# Create new digital newspapers archive header

## What are you doing in this PR?

Creating the new design of the header that lives on theguardian.newspapers.com. 



[**Trello Card**](https://trello.com/c/pr57zkkZ/1099-build-layout-for-newspaper-archivecom-header)
[Figma](https://www.figma.com/design/h53HMhA753GLQ4ycN1uoPc/newspaper-archive?node-id=1606-23957&node-type=canvas&t=MFmOXb7ZliDlqT41-0)



## How to test

Go to `https://support.theguardian.com/newspaper-archive-header`. I've tested this as an iframe by changing the DEV.public.conf to allow another site and embedding the iframe there.

Or see in storybook.




## Screenshots
Mobile
![image](https://github.com/user-attachments/assets/2c98a60c-c645-4bbe-86cc-530c28f1c64b)

Tablet
![image](https://github.com/user-attachments/assets/ea56cde4-2aa9-4467-95ff-3870f8bc7a68)

Wide
![image](https://github.com/user-attachments/assets/05f95153-dda3-40aa-8270-c679011df021)
